aiven/resource_service_user.go

```diff
@@ -68,6 +68,8 @@ var aivenServiceUserSchema = map[string]*schema.Schema{
 		Type:        schema.TypeString,
 		Sensitive:   true,
 		Computed:    true,
+		Optional:    true,
+		ForceNew:    true,
```

Comment:
Why is `ForceNew` set to `true`? Code seems to update existing user so shouldn't be necessary. This would be problematic for Kafka users who have both certificate and password and this would cause the certificate to be changed as well.

Reply:
We do not have the `update` method for `service_user` resource, and all fields should be either `computed` or `ForceNew`. 

`create` operation does two API calls: `POST` to create a user and then `PUT` to update the password. Still, both of them are happening sort of in one transaction, and subsequent `read` operations should get a consistent certificate`s output.

Reply:
Ah, didn't read where the code actually was. Still, shouldn't we rather add the update method and make anything that cannot be updated have `ForceNew: true` instead of the one thing that actually does support update causing new user to be created?

Reply:
Sure, let's do that.